src/components/ResultItem/ResultItem.tsx

```diff
@@ -1,17 +1,43 @@
 import React from 'react';
 
+type meta = {
+  type?: string;
+  date?: Date;
+  lastUpdated?: Date;
+};
+
 type ResultItemProps = {
   className?: string;
   href: string;
-  meta?: string;
+  meta?: meta;
   text?: string;
   title: string;
 };
 
-export const ResultItem = ({ meta, title, href, text }: ResultItemProps) => (
+export const ResultItem = ({
+  meta: { type, date, lastUpdated },
+  title,
+  href,
+  text
+}: ResultItemProps) => (
   <div className="result-item">
     <div className="result-item__meta">
-      <span className="result-item__meta-type">{meta}</span>
+      <span className="result-item__meta-type">{type}</span>
+      <span className="result-item__meta-type--date">
+        {new Intl.DateTimeFormat('en-GB', {
+          year: 'numeric',
+          month: 'long',
+          day: '2-digit'
+        }).format(date)}
```

Comment:
Nice - though I imagine we will probably pass the date prop through as a string for now. (depends on what Solr sends back)

Reply:
You can abstract this and following instance into separate IntlDate component (or similar name).

I presume Solr returns a string (simpler format) which could accept string or date.

You could always accept both formats and check if it's typeof string then convert to a Date object before converting into Intl.Date